tests/orm/nodes/data/test_data.py

```diff
@@ -109,6 +109,25 @@ def _generate_class_instance(data_class):
             )
             return instance
 
+        if (
+            data_class is orm.AbstractCode
+            or data_class is orm.Code
+            or data_class is orm.InstalledCode
+            or data_class is orm.ContainerizedCode
+            or data_class is orm.PortableCode
+        ):
+            # Taken from `test_code.py`
+            import uuid
+
+            computer = orm.Computer(
+                label=uuid.uuid4().hex,
+                transport_type='core.local',
+                hostname='localhost',
+                scheduler_type='core.slurm',
+            ).store()
+            instance = orm.Code(remote_computer_exec=(computer, '/bin/cat'))
+            return instance
```

Comment:
This is then tested in the `test_data_exporters` function right?

Reply:
Good question, I didn't even look that far... I just implemented the instance generator so that the tests don't fail 😅 but yes, indeed, it's used in `test_data_exporters`. Though, that function actually only checks for the correct return types of `_exportcontent`.